config/routes.rb

```diff
@@ -16,6 +16,7 @@
   get 'privacy', to: 'pages#privacy'
   get 'find-placecal', to: 'pages#find_placecal'
   get 'our-story', to: 'pages#our_story'
+  get 'terms-of-use', to: 'pages#terms_of_use', as: :terms_of_use
```

Comment:
Why does this one require the `, as:` clause whereas others do not?

Reply:
ugh no idea, weird